modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc

```diff
@@ -45,45 +49,49 @@ The only mandatory parameter for the script is the newly configured IP address o
     You also need to provide configuration settings for reverse name resolution.
     Changing network settings is done in the same way as with renaming any other system.
 . Reboot the {productname} Server to use the new network configuration and to ensure the hostname has changed.
-. Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
-    If the server is not using the new hostname, the script fails.
-    Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
-
-. *Skip this step if the clients are managed via a {productname} proxy.*
-  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
-  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server) (the filename is [path]``/etc/venv-salt-bundle/minion``or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``):
+. On the container host, from the command line, execute the following command to configure the public IP address of the server:
 +
 ----
-master: <new_hostname>
+mgrctl exec -ti -- spacewalk-hostname-rename <PUBLIC_IP_ADDRESS>
 ----
-
-. Restart the [systemitem]``venv-salt-minion`` service (if using the {salt} bundle):
++
+If the server is not using the new hostname, the script fails.
+  Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
+. *Skip this step if the clients are managed via a {productname} proxy.*
+  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
+  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server).
+  The filename is [path]``/etc/venv-salt-bundle/minion`` or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``:
 +
 ----
-systemctl restart venv-salt-minion
+master: <new_hostname>
 ----
 
-+
-Or  the [systemitem]``salt-minion``  service (if not using the {salt} bundle):
+. Restart the server.
+  On the container host, execute the restart command:
 +
 ----
-systemctl restart salt-minion
+mgradm restart
```

Comment:
I did not notice this on the original PR, but this does not make sense.
Why in the one procedure we are mixing SMLM server restart and changing salt config of salt clients?
I think the restart step was not a restart step of the SMLM server, but salt client, so the `systemctl restart venv-salt-minion` should be there instead of `mgradm restart`.

I have no experience renaming the server, but this is confusing.

At least there should be two procedures -> one for changes on the SMLM server and another what to do with directly connected salt clients.

Reply:
I ported fixes from master